addons/l10n_ec_website_sale/tests/test_l10n_ec_website_sale.py

```diff
@@ -1,16 +1,24 @@
 from odoo.tests.common import HttpCase, tagged
-from odoo.addons.l10n_ec_edi.tests.common import TestEcEdiCommon
 
 
 @tagged('post_install_l10n', 'post_install', '-at_install')
-class TestUi(HttpCase, TestEcEdiCommon):
+class TestUi(HttpCase):
 
     def test_checkout_address_ec(self):
+        company = self.env['res.company'].create({
+            'name': "EC Test Company",
+            'vat': "1792366836001",
+            'street': "Avenida Machala 42",
+            'zip': "090514",
+            'city': "Guayaquil",
+            'country_id': self.env.ref('base.ec').id,
+        })
+        self.env.company = company
```

Comment:
Could be onelined?

Reply:
I'd rather not as it would result in a ~= 210 column line.

Reply:
I'm stupid ahhah, it's fixed!